src/bin/pg_autoctl/string_utils.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * src/bin/pg_autoctl/string_utils.c
+ *   Implementations of utility functions for string handling
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the PostgreSQL License.
+ *
+ */
+
+#include <errno.h>
+#include <inttypes.h>
+#include <stdlib.h>
+
+#include "snprintf.h"
+#include "string_utils.h"
+
+/*
+ * intToString converts an int to an IntString, which contains a decimal string
+ * representation of the integer.
+ */
+IntString
+intToString(int64_t number)
+{
+	IntString intString;
+
+	intString.intValue = number;
+
+	pg_snprintf(intString.strValue, INTSTRING_MAX_DIGITS, "%" PRId64, number);
```

Comment:
Let's keep this sprintf, beacuse it should be changed to `sformat` later.

```suggestion
	sprintf(intString.strValue, INTSTRING_MAX_DIGITS, "%" PRId64, number);
```

Reply:
done